ModAssistant/App.xaml.cs

```diff
@@ -168,12 +168,12 @@ private async Task ArgumentHandler(string[] args)
                         }
                         else
                         {
-                            OneClickInstaller.Register(args[1], true);
+                            OneClickInstaller.Register(args[1], true, args[2]);
                         }
 
                         Update = false;
                         GUI = false;
-                        args = Shift(args, 2);
+                        args = Shift(args, args.Length);
```

Comment:
`args.Length` will cause the rest of the arguments to get discarded. Avoid arbitrary Shifts like that. Most arguments are meant to be called by Mod Assistant itself, so instead make sure those are consistent with a set number of Shifts.

Reply:
Thanks for the clarification. Wasn't sure if the arguments were ever used outside of MA calling itself. PR has been updated.